pytest_hoverfly/base.py

```diff
@@ -73,7 +73,7 @@ def get_container(
     container_name: t.Optional[str] = None,
     ports: t.Optional[t.Dict[str, t.Optional[t.List[t.Dict[str, int]]]]] = None,
     image: str = IMAGE,
-    timeout: float = 3.0,
+    timeout: float = 30.0,
```

Comment:
Isn't enough to update the timeout in consumers? Why do we need to increase default value?

Reply:
@Se7ge reverted.